jspsych.js

```diff
@@ -840,6 +840,9 @@ var jsPsych = (function() {
     $('#jspsych-progressbar-inner').css('width', progress.percent_complete + "%");
   }
 
+  //Leave a trace in the DOM that jspsych was loaded
+  $("html").attr("jspsych", "present");
```

Comment:
I'm going to try and remove jQuery as a dependency in the near future, so rewriting this line would be great.